bbc/simorgh
# Arabic and Mundo nav update

Resolves JIRA [number]

Overall changes
======
Updates to BBC Arabic and Mundo navigation bar ahead of their Front page migrationt to Tipo

Code changes
======
- edited the navigation section of src/app/lib/config/services/arabic.ts
- edited the navigation section of src/app/lib/config/services/mundo.ts to delete the last link 
- updated snapshots
- 


Testing
======
1. Open BBC Arabic Front page, the nav bar should containt just nine items, none should 404 
2. Open BBC Mundo Front page the last list on the navigation should be Centroamérica Cuenta

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
